resources/bash/comment.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+#only comments
+x=3
+#comment
+if (($de -a $en)); then #comment
+echo "helo" #comment
+fi #comment
+: '
+This is a multiline comment.
+Its created by using the colon (:) as a no-op command,
+followed by a block enclosed in single quotes.
+This block is treated as a string literal and is ignored.
```

Comment:
Is this a multiline comment now or not? The test case says it should not count as comment?